s3/pkg/datastore/yig/common/constants.go

```diff
@@ -0,0 +1,5 @@
+package common
+
+const (
+	MIN_PART_SIZE = 128 << 10 // 128KB
```

Comment:
please add to constants and remove magic numbers from code

Reply:
MIN_PART_SIZE is the constant, do you mean that we need to move '128 << 10' to another constant?